src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,152 @@
+package core.basesyntax;
+
+enum Color {
```

Comment:
```suggestion
public enum Color {
```
Make all classes **public**